src/main/java/uk/gov/justice/hmpps/prison/aop/LoggingAspect.java

```diff
@@ -26,12 +28,12 @@ public void loggingPointcut() {
 
     @AfterThrowing(pointcut = "loggingPointcut()", throwing = "e")
     public void logAfterThrowing(final JoinPoint joinPoint, final Throwable e) {
-        log.error(
-                String.format(
-                        "Exception in pointcut %s.%s()",
-                        joinPoint.getSignature().getDeclaringTypeName(),
-                        joinPoint.getSignature().getName()),
-                e);
+        if (!(e instanceof EntityNotFoundException || e instanceof NoContentException)) {
```

Comment:
These two account for 14,000 exceptions in the logs in the last 24 hours, so remove from here and log message instead in controller advise